openprescribing/matrixstore/build/import_prescribing.py

```diff
@@ -166,8 +171,7 @@ def format_as_sql_rows(matrices, connection):
         cursor.execute(
             'INSERT OR IGNORE INTO presentation (bnf_code) VALUES (?)',
             [row.bnf_code])
-        # Crude progress logging
-        print(row.bnf_code)
+        logger.info('Writing data for %s', row.bnf_code)
```

Comment:
This is going to dominate the output.  If we think it possible that there would be errors in writing data for a specific BNF code, we should keep it.  Otherwise, if this is just here for progress logging, could we just log something like: `"Written X000/82033 BNF codes"`?